<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>601</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;SmartDiff&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Step 4: Confirm your input&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="FuncInput_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>491</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string extracomment="default is 0"/>
   </property>
   <property name="placeholderText">
    <string extracomment="Default is 0"/>
   </property>
  </widget>
  <widget class="QPushButton" name="OKButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>450</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="FuncLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Differentiate function&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="FirstLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>180</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1st component</string>
   </property>
  </widget>
  <widget class="QLabel" name="atLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;at&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="xVal_1">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="xVal_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>310</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="xVal_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>340</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="FuncInput_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>491</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string extracomment="default is 0"/>
   </property>
   <property name="placeholderText">
    <string extracomment="Default is 0"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="FuncInput_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>491</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string extracomment="default is 0"/>
   </property>
   <property name="placeholderText">
    <string extracomment="Default is 0"/>
   </property>
  </widget>
  <widget class="QLabel" name="secondLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>210</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>2nd component</string>
   </property>
  </widget>
  <widget class="QLabel" name="thirdLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>240</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>3rd component</string>
   </property>
  </widget>
  <widget class="QLabel" name="xLabel_1">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>x1 =</string>
   </property>
  </widget>
  <widget class="QLabel" name="xLabel_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>310</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>x2 =</string>
   </property>
  </widget>
  <widget class="QLabel" name="xLabel_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>x3 =</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>390</y>
     <width>191</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Show function value</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
